src/progress/progress_set.rs

```diff
@@ -776,14 +809,12 @@ mod test_progress_set {
 
     #[test]
     fn test_membership_change_configuration_empty_sets() {
-        assert!(check_membership_change_configuration((vec![], vec![]), (vec![], vec![])).is_err())
+        assert!(check_membership_change_configuration((vec![], vec![]), (vec![], vec![])).is_ok())
     }
 
     #[test]
     fn test_membership_change_configuration_empty_voters() {
-        assert!(
-            check_membership_change_configuration((vec![1], vec![]), (vec![], vec![]),).is_err()
-        )
+        assert!(check_membership_change_configuration((vec![1], vec![]), (vec![], vec![]),).is_ok())
```

Comment:
/cc @Hoverbear PTAL

Reply:
It's because currently an empty configuration is allowed as target. Do you think we need to ban it? @Hoverbear

Reply:
I'm not really sure it's a valid state.

Reply:
I guess it's valid means there is a machanism to remove all peers from a Raft group, although TiKV doesn't need it.